package.json

```
@@ -6,13 +6,13 @@
   "scripts": {
     "test": "npm run cypress:lint && npm run cypress:run",
     "build": "webpack --mode production",
-    "dev": "INSTRUMENTATION=1 RAND=1 webpack-dev-server --mode development",
+    "dev": "cross-env INSTRUMENTATION=1 RAND=1 webpack-dev-server --mode development",
     "start": "http-server -p 8080",
     "banana": "banana-checker i18n/",
     "adb:connect": "adb forward tcp:6000 localfilesystem:/data/local/debugger-socket",
-    "b2g:uninstall": "./node_modules/b2g-cli-tool/b2g.js uninstall Wikipedia",
-    "b2g:install": "./node_modules/b2g-cli-tool/b2g.js install .",
-    "deploy": "INSTRUMENTATION=1 RAND=1 npm run build && npm run adb:connect && ( npm run b2g:uninstall; npm run b2g:install )",
+    "b2g:uninstall": "node ./node_modules/b2g-cli-tool/b2g.js uninstall Wikipedia",
+    "b2g:install": "node ./node_modules/b2g-cli-tool/b2g.js install .",
+    "deploy": "cross-env INSTRUMENTATION=1 RAND=1 npm run build && npm run adb:connect && ( npm run b2g:uninstall || echo ) && npm run b2g:install",
```

```suggestion
    "deploy": "cross-env INSTRUMENTATION=1 RAND=1 npm run build && npm run adb:connect && npm run b2g:uninstall & npm run b2g:install",
```

we can also do this and let uninstall running in background, but I afraid this will bring some unexpected result.

anyone experience with this?